_data/entertainment.yml

```diff
@@ -68,36 +75,42 @@ websites:
     - name: Plex
       url: https://www.plex.tv
       twitter: plexapp
+      facebook: TVPlex
```

Comment:
Their website links to the [plexapp](https://www.facebook.com/plexapp) Facebook page. Where did you find [TVPlex](https://www.facebook.com/TVPlex)?